pkg/flare/flare.go

```diff
@@ -13,6 +13,6 @@ import (
 
 // SendFlare sends a flare and returns the message returned by the backend. This entry point is deprecated in favor of
 // the 'Send' method of the flare component.
-func SendFlare(archivePath string, caseID string, email string) (string, error) {
-	return helpers.SendTo(archivePath, caseID, email, config.Datadog.GetString("api_key"), utils.GetInfraEndpoint(config.Datadog))
+func SendFlare(archivePath string, caseID string, email string, source string) (string, error) {
```

Comment:
Not sure if it would be useful to add a brief comments of what some of the sources values could be expected.